src/reconcile/rec_visibility.c

```diff
@@ -463,10 +463,6 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
      */
     if (__rec_need_save_upd(
           session, upd_select->upd, max_txn, max_ts, list_uncommitted, r->flags)) {
-        /* During recovery, there are no transaction id's. */
-        WT_ASSERT(
-          session, !F_ISSET(S2C(session), WT_CONN_RECOVERING) ? r->max_txn != WT_TS_NONE : true);
-
```

Comment:
This assert is wrong if we have skipped writing the new image and cleared the time pairs on the old disk image when we doing verify.